src/OrchardCore.Modules/OrchardCore.Title/ViewModels/TitlePartSettingsViewModel.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using OrchardCore.Title.Models;
+
+namespace OrchardCore.Title.ViewModels
+{
+    public class TitlePartSettingsViewModel
+    {
+        public bool AllowCustomTitle { get; set; } = true;
+        public string Pattern { get; set; }
```

Comment:
It not setting correctly maybe related to this `AllowCustomTitle = true` line

Reply:
ViewModel should not set a default value. It's on the model that you should have it.

Reply:
Artifact of my testing. Removed.